### PR TITLE
(DO NOT MERGE, REFERENCE ONLY) feat: add rewACT bolt-on reward model for ACT

### PR DESCRIPTION
**Pull request by villekuosmanen**
## What this does

Explain what this PR does. Feel free to tag your PR with the appropriate label(s).

Examples:
| Title | Label |
|----------------------|-----------------|
| Fixes #[issue] | (🐛 Bug) |
| Adds new dataset | (🗃️ Dataset) |
| Optimizes something | (⚡️ Performance) |

## How it was tested

Explain/show how you tested your changes.

Examples:

- Added `test_something` in `tests/test_stuff.py`.
- Added `new_feature` and checked that training converges with policy X on dataset/environment Y.
- Optimized `some_function`, it now runs X times faster than previously.

## How to checkout & try? (for the reviewer)

Provide a simple way for the reviewer to try out your changes.

Examples:

```bash
pytest -sx tests/test_stuff.py::test_something
```

```bash
lerobot-train --some.option=true
```

## SECTION TO REMOVE BEFORE SUBMITTING YOUR PR

**Note**: Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR. Try to avoid tagging more than 3 people.

**Note**: Before submitting this PR, please read the [contributor guideline](https://github.com/huggingface/lerobot/blob/main/CONTRIBUTING.md#submitting-a-pull-request-pr).

### CODE REVIEW

File: src/lerobot/configs/policies.py [hunk 1]
```diff
@@ -63,7 +63,7 @@ class PreTrainedConfig(draccus.ChoiceRegistry, HubMixin, abc.ABC):
     # automatic gradient scaling is used.
     use_amp: bool = False
 
-    push_to_hub: bool = True
+    push_to_hub: bool = False
```
Comment (villekuosmanen) [inline src/lerobot/configs/policies.py:66]: not a relevant change

File: src/lerobot/datasets/utils.py [hunk 2]
```diff
@@ -268,7 +268,7 @@ def hf_transform_to_torch(items_dict: dict[torch.Tensor | None]):
         elif first_item is None:
             pass
         else:
-            items_dict[key] = [x if isinstance(x, str) else torch.tensor(x) for x in items_dict[key]]
+            items_dict[key] = [x if isinstance(x, str) or (isinstance(x, list) and isinstance(x[0], str)) else torch.tensor(x) for x in items_dict[key]]
```
Comment (villekuosmanen) [inline src/lerobot/datasets/utils.py]: also not a relevant change, you won't need this.

**Comment by helper2424** [comment]
Nice, we’ve already done a similar exercise with HIL-SERL. We implemented the reward model as a standalone component (see https://github.com/huggingface/lerobot/tree/main/src/lerobot/policies/sac/reward_model).
It was quite straightforward—the only nuance was integrating the reward model into the RL workflow, which fit in naturally.
